frontend/packages/lint-config/README.md

```diff
@@ -46,4 +47,29 @@ import cdoEslint from './eslint/node.mjs';
 
 /** @type {import('eslint').Linter.Config[]} */
 export default [...cdoEslint];
-```
\ No newline at end of file
+```
+
+## Prettier Configuration
+
+Code.org uses the default prettier configuration with some overrides in the `prettier` directory.
+
+To use the shared configuration, add the following key to `package.json`:
+
+```
+  "prettier": "@code-dot-org/lint-config/prettier/index.mjs",
+```
+
+The Code.org prettier configuration also does not utilize ESLint/Prettier plugins in accordance to the
+[Prettier](https://prettier.io/docs/en/integrating-with-linters) recommendation. Instead, the `lint` and
+`lint:fix` scripts run `prettier --check .` and `prettier --write .` respectively. Ensure the following scripts
+are added in the consuming packages' `package.json`:
+
+```
+    "lint": "eslint . && prettier --check .",
+    "lint:fix": "eslint --fix . && prettier --write .",
+```
+
+### Editor Integration
+
+Since prettier will not error out as a linter error, be sure to configure your editor of choice to automatically
+run prettier on save. For instructions for your editor of choice, see the [Prettier documentation](https://prettier.io/docs/en/editors).
```

Comment:
This concerns me a bit because apps/ uses eslint and, when I accidentally had prettier as the default formatter a few months, it didn't work correctly. Because these are in the same repo, it would be a big pain to have to switch default formatters sounds like a big pain

Reply:
From the contributor's perspective, this process should be completely transparent and handled automatically (and won't result in the red squiggly for format errors).

When you work on the `apps` directory, the editor will apply the prettier configuration from `apps/.prettierrc.js` and when you work on `frontend`, the prettier config defined in `package.json` will be applied. So really the contributor just formats (or configures to automatically format on save) and the correct config is 

I have a video showing this behavior:

[Screencast From 2024-11-26 11-06-17.webm](https://github.com/user-attachments/assets/5ef919dd-ce1a-448e-80bf-604b0940c5d4)

Reply:
Based on our offline conversation, we were able to configure VS Code to work and we'll refine the prettier setup over the next few weeks to ensure we get a good balance in our linter toolset. 

Also, we'll need to circle back to add a pre-commit hook, task [here](https://codedotorg.atlassian.net/browse/ACQ-2851).